stable/cluster-overprovisioner/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: "3.1"
```

Comment:
I'm happy to remove this change, if you prefer.

Reply:
I think it's good.